apps/dashboard/app/apps/ood_app.rb

```diff
@@ -313,7 +313,8 @@ def version
                   elsif version_from_file.to_s.downcase == 'unknown'
                     nil
                   else
-                    (version_from_file || version_from_git).strip
+                    # Both could be null. Added null safety
+                    (version_from_file || version_from_git)&.strip
```

Comment:
Does casting to string here do the same? I think that's the more common pattern.

```ruby
(version_from_file || version_from_git).to_s.strip
```

Reply:
That was my first solution, but it breaks the interface.
It returns empty string instead of `nil`

If you think empty string is ok, I will update to the fix and update the tests

Reply:
Got it, yea return values matter here. Thanks.

> If you think empty string is ok,

No, this'll be fine.